src/CameraWidgetForPhoneGap/CameraWidgetForPhoneGap.webmodeler.js

```diff
@@ -18,16 +18,16 @@ module.exports.preview = React.createClass({
             this.alignment = "horizontal";
         }
 
-        var divPreview = React.DOM.div({ className: `${this.widgetClass}-preview ${this.props.imageContainerClass} ${this.widgetClass}-${this.alignment}` });
-        var button = React.DOM.button({ className: `btn btn-primary ${this.widgetClass}-button  ${this.props.buttonClass} ${this.widgetClass}-${this.alignment}` }, this.props.buttonText);
+        var divPreview = React.DOM.div({ className: this.widgetClass + "-preview" + this.props.imageContainerClass + this.widgetClass + "-" + this.alignment });
+        var button = React.DOM.button({ className: "btn btn-primary" + this.widgetClass + "-button" + this.props.buttonClass + this.widgetClass + "-" + this.alignment }, this.props.buttonText);
         var elements = [ divPreview, button ];
 
         if (/below|right/.test(this.props.imageLocation)) {
             elements.reverse();
         }
 
         return React.DOM.div(
-            { className: "widget-google-maps" },
+            { className: "widget-camera-phonegap" },

```

Comment:
Class name does not match any classes in the widget `css` file there exists `wx-CameraWidgetForPhoneGap`